RealmSwift/Combine.swift

```diff
@@ -1403,7 +1421,7 @@ public enum RealmPublishers {
 
         /// :nodoc:
         public func receive<S>(subscriber: S) where S: Subscriber, S.Failure == Never, Output == S.Input {
-            let token = self.collection.observe(on: self.queue) { change in
+            let token = self.collection.observe(keyPaths: nil, on: self.queue) { change in
```

Comment:
These shouldn't require the new parameter to be added.

Reply:
Both "keyPaths", and "on [queue]" appear required by the compiler because I'm unable to add a default value to protocol method. Is there another way?

<img width="1113" alt="Screen Shot 2021-07-27 at 1 25 20 PM" src="https://user-images.githubusercontent.com/25092858/127199866-a1bc46fa-ba65-40d7-b884-1d5cea738505.png">

Reply:
Ah, right, you can't set default parameter values on protocol functions. That unfortunately means that changing the existing function on the protocol is a breaking change, and for now we need to add an overload to the protocol instead.

Reply:
If we overload the protocol, will this mean that all the realm collection types will also need two types observe functions?

Reply:
Or is there a way a single stub can satisfy both protocol requirements? I'm asking because so far the changes I'm making locally are getting messy compared to just adding the single default parameter.

Reply:
I think it does require two separate functions unfortunately. Swift theoretically could make `func foo(value: Int = 0)` work for a protocol requirement of `func foo()`, but it doesn't.